csvw/csvw_test.go

```diff
@@ -42,6 +43,38 @@ func TestNew(t *testing.T) {
 	})
 }
 
+func TestNewCustom(t *testing.T) {
+	Convey("Given a complete metadata struct", t, func() {
+		time := time.Now()
+		m := &dataset.Metadata{
+			Version: dataset.Version{
+				ReleaseDate: time.Format("01-02-2006 15:04:05"),
+				Dimensions: []dataset.VersionDimension{
+					{
+						Label: "Label 1",
+					},
+					{
+						Label: "Label 2",
+					},
+				},
+			},
+			DatasetDetails: dataset.DatasetDetails{
+				Title: "Label 1 and Label 2",
+			},
```

Comment:
I think the function does not read this value and I wouldn't expect it in the metadata input. It seems we use it for the title assertion but that is generated based on the dimensions so a hardcoded value there would be acceptable.

Similarly with the release date.